jerry-core/parser/regexp/re-compiler.c

```diff
@@ -388,7 +388,7 @@ re_parse_alternative (re_compiler_ctx_t *re_ctx_p, /**< RegExp compiler context
 
         ECMA_TRY_CATCH (empty,
                         re_parse_char_class (re_ctx_p->parser_ctx_p,
-                                             re_append_char_class,
+                                             (re_char_class_callback) re_append_char_class,
```

Comment:
@franc0is, good point. @robertsipka, please change `typedef void (*re_char_class_callback) (void *re_ctx_p, uint32_t start, uint32_t end);` to `typedef void (*re_char_class_callback) (void *re_ctx_p, ecma_char_t start, ecma_char_t end);` and remove the cast.